inputreader_windows.go

```diff
@@ -38,11 +38,16 @@ func newInputReader(r io.Reader) (cancelreader.CancelReader, error) {
 		return fallback(r)
 	}
 
-	originalMode, err := prepareConsole(conin,
-		windows.ENABLE_MOUSE_INPUT,
+	modes := []uint32{
 		windows.ENABLE_WINDOW_INPUT,
 		windows.ENABLE_EXTENDED_FLAGS,
-	)
+	}
+
+	if enableMouse {
```

Comment:
I think would be good to add why enableMouse isn't default, other I feel tends to move back modes initilization.

What do you think?

Reply:
added a comment

Reply:
thank you!